source/bson-corpus/bson-corpus.rst

```diff
@@ -338,6 +338,27 @@ Implementations MAY ignore or modify them to match legacy treatment of
 deprecated types.  The ``converted_bson`` and ``converted_extjson`` fields MAY
 be used to test conversion to a standard type or MAY be ignored.
 
+Prose Tests
+===========
+
+The following tests have not yet been automated, but MUST still be tested.
+
+1. Prohibit null bytes in null-terminated strings when encoding BSON
```

Comment:
My intended meaning of "encoding BSON" was "producing raw BSON bytes".

As it pertains to libbson, I think this would only be possible via the `bson_append_*` functions since the key `char *` and length are solicited as separate parameters. So, what would happen if `strlen(key) != `key_length`? The `BSON_APPEND_*` macros compute the length with `strlen()`, so that wouldn't be possible there.

Reply:
I'll amend the spec to clarify what I mean by "encode BSON" and resolve this when I do.

Reply:
I added clarification about "encoding BSON" in the first paragraph within this section.